pallets/payment/src/lib.rs

```diff
@@ -40,7 +40,14 @@ pub mod pallet {
 	pub type AssetIdOf<T> =
 		<<T as Config>::Asset as MultiCurrency<<T as frame_system::Config>::AccountId>>::CurrencyId;
 	pub type BoundedDataOf<T> = BoundedVec<u8, <T as Config>::MaxRemarkLength>;
+	/// type of ScheduledTask used by the pallet
 	pub type ScheduledTaskOf<T> = ScheduledTask<<T as frame_system::Config>::BlockNumber>;
+	/// list of ScheduledTasks, stored as a BoundedBTreeMap
+	pub type ScheduledTaskList<T> = BoundedBTreeMap<
+		(<T as frame_system::Config>::AccountId, <T as frame_system::Config>::AccountId),
+		ScheduledTaskOf<T>,
+		<T as Config>::MaxRemarkLength,
```

Comment:
`MaxRemarkLength` is a number of bytes and this is a bound for a number of items, might be confusing/misleading to use the same constant that at least would have to be named differently. It's reasonable to expect people configuring them very differently like a remark of 8 bytes but allowing to schedule 1000 tasks or who knows, better to introduce a new one ;)

Reply:
Oops, forgot to change it